config/envs/development.py

```diff
@@ -21,4 +21,8 @@ class DevelopmentConfig(DefaultConfig):
             ("Sunderland City Council", "Worcester City Council"),
             ("Sunderland City Centre", "Blackfriars - Northern City Centre", "Worcester"),
         ),
+        "test.communities.gov.uk": (
+            ("Sunderland City Council", "Worcester City Council"),
+            ("Sunderland City Centre", "Blackfriars - Northern City Centre", "Worcester"),
+        ),
```

Comment:
How come we need this?

Reply:
This is the domain used by the Test AD, so it's a convenience for users that have Test AD accounts.